src/instructlab/configuration.py

```diff
@@ -367,7 +367,7 @@ class _evaluate(BaseModel):
     base_model: str
     branch: Optional[str] = None
     base_branch: Optional[str] = None
-    gpus: Optional[int] = 1
+    gpus: Optional[int] = None
```

Comment:
I think None needs to be allowed here as well: https://github.com/instructlab/instructlab/blame/main/src/instructlab/model/evaluate.py#L279